darwin/cli.py

```diff
@@ -22,61 +35,78 @@ def main():
 
     # Authenticate user
     if args.command == "authenticate":
-        email = input("Username (email address): ")
-        password = getpass.getpass(prompt="Password: ", stream=None)
-        projects_dir = prompt("Project directory", "~/.darwin/projects")
-        f.authenticate(email, password, projects_dir)
-        print("Authentication succeeded. ")
+        api_key = getpass.getpass(prompt="API key: ", stream=None)
+        if api_key.strip() == "":
+            print(
+                "API Key needed, generate one for your team: https://darwin.v7labs.com/?settings=api-keys"
+            )
+            return
+        default_team = input("Make this team default? [y/N] ") in ["Y", "y"]
+        datasets_dir = prompt("Datasets directory", "~/.darwin/datasets")
+        f.authenticate(api_key, datasets_dir, default_team)
+        print("Authentication succeeded.")
 
     # Select / List team
     elif args.command == "team":
         if args.team_name:
             f.set_team(args.team_name)
-        elif args.list:
-            f.list_teams()
-        else:
+        elif args.current:
             f.current_team()
-
-    # Create new project
-    elif args.command == "create":
-        f.create_dataset(args.project_name)
+        else:
+            f.list_teams()
 
     # List existing projects
     elif args.command == "local":
         f.local()
 
-    # Print projects local path
-    elif args.command == "path":
-        path = f.path(args.project_name)
-        print(path)
-
-    # Print the url of a remote project
-    elif args.command == "url":
-        url = f.url(args.project_name)
-        print(url)
-
     # Download dataset
     elif args.command == "pull":
```

Comment:
should we not have this inside the elif args.command == "dataset" below to match the API of other operations such as push, create remove and so on?

Reply:
only local and pull are left, but that's because I haven't implemented them, waiting for andrea to finish the new export code.